citystreets/urls.py

```diff
@@ -24,3 +26,6 @@
     path("map/", include("map.urls")),
     path("location/", include("location.urls")),
 ]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Could you explain why we add MEDIA_URL only in the DEBUG environment?

Reply:
I read from the django docs (don't exactly remember where) that it is not safe to do so in production env.

Reply:
https://stackoverflow.com/a/5518073

Reply:
Thank you for the [link](https://stackoverflow.com/questions/5517950/django-media-url-and-media-root/5518073#5518073). It's no longer needed!